.github/workflows/ci.yml

```diff
@@ -22,7 +24,7 @@ jobs:
       - name: Build contracts
         run: |
           forge --version
-          forge build --sizes
+          forge build
```

Comment:
It's disappointing that we have to remove this for *all* clients just because Comet is over the size limit. We should create an issue to track this. Options to add this back are:
1. Get comet below the size limit
2. Delete the comet contracts, then build, then restore them

Reply:
I am surprised we can't ignore specific contracts in the `forge build` command

Reply:
https://github.com/ScopeLift/flexible-voting-clients/pull/1/commits/7cb815322966c9495f5f4a081b4f2a27c53fbf99

Reply:
LGTM, unresolved for related comment: Where are we tracking that the Comet contract is above the size limit? My suggestion would be to create an issue for it, and in that issue mention that once it's below the size limit we should undo this change of excluding Comet

Reply:
https://github.com/ScopeLift/flexible-voting-clients/issues/6